forage-android/src/main/java/com/joinforage/forage/android/ui/ForagePANEditText.kt

```diff
@@ -27,9 +32,16 @@ class ForagePANEditText @JvmOverloads constructor(
     context: Context,
     attrs: AttributeSet? = null,
     defStyleAttr: Int = R.attr.foragePanEditTextStyle
-) : LinearLayout(context, attrs, defStyleAttr), TextWatcher, ActionMode.Callback {
+) : ForageUI, LinearLayout(context, attrs, defStyleAttr), TextWatcher, ActionMode.Callback {
     private val textInputEditText: TextInputEditText
     private val textInputLayout: TextInputLayout
+    private val manager: PanElementStateManager = PanElementStateManager.forEmptyInput()
+    override var isValid: Boolean = manager.getState().isEmpty
+        get() = manager.getState().isEmpty
```

Comment:
`isValid` seems to be mapping to `isEmpty`

Reply:
nice catch! Fixed in [396ff6a](https://github.com/teamforage/forage-android-sdk/pull/55/commits/396ff6aad67733f049b437435e51baa8385f0dee)